armi/reactor/blueprints/assemblyBlueprint.py

```diff
@@ -175,6 +175,9 @@ def _constructAssembly(self, cs, blueprint):
         aziMeshPoints = self.azimuthalMeshPoints or 1
         a.p.AziMesh = aziMeshPoints
 
+        # TODO: How can this be done better?
```

Comment:
Let's not commit a `TODO` to the repo.

Reply:
The changes in this file have been removed altogether with so no more `TODO`. 
https://github.com/terrapower/armi/pull/993/commits/9107822341523e8c5f8e27f6e77f4d5e01fdca98